### ISSUE TITLE
NUnit 3.9 only available as nupkg, but nuget can't be used on C++/CLI projects

### ISSUE THREAD

**Issue by erict-powersoft** [nunit/nunit#2691]
I have a C++/CLI based test bed which I used successfully with NUnit 2.6.4 on VS2010.  I've switched now to  VS2015, and NUnit 2.6.4 doesn't like me anymore.  I'm now trying to ugprade to NUnit 3.9 but have been fighting with Nuget and the error message that it can't install to target 'native, Version=v0.0'.  the easiest thing would be to use 3.9 like I used 2.6.4, but there's no MSI to install...

**Comment by ChrisMaddock** [comment]
In each framework release, we provide a plain .zip file containing the dlls. Unfortunately however, that artifact was missed for 3.9 (see https://github.com/nunit/nunit/issues/2646).

I suggest you access the dll's by unzipping the NuGet package for 3.9, and then utilising the zip distribution for future releases. Would that solve your problem?

On the msi - we stopped distributing a framework msi due to a perceived lack of demand - I think you're only the second person to ask this, since we stopped it a couple of years ago! There's still an msi available for the console runner, if you wish to use that.
